Modules/Loadable/SubjectHierarchy/Widgets/qSlicerSubjectHierarchyPluginLogic.cxx

```diff
@@ -123,7 +125,9 @@ qSlicerSubjectHierarchyPluginLogic::~qSlicerSubjectHierarchyPluginLogic() = defa
 void qSlicerSubjectHierarchyPluginLogic::registerCorePlugins()
 {
   qSlicerSubjectHierarchyPluginHandler::instance()->registerPlugin(
-    new qSlicerSubjectHierarchyFolderPlugin());
+    new qSlicerSubjectHierarchyViewContextMenuPlugin());
+  qSlicerSubjectHierarchyPluginHandler::instance()->registerPlugin(
+    new qSlicerSubjectHierarchyViewContextMenuPlugin());
```

Comment:
The new view context menu plugin seems to be registered twice, but the folder plugin is now missing.

Reply:
Thank you, good catch! I've noticed this and fixed locally just haven't pushed the change yet, as I'm still working on adding some of the actions that @pieper recommended - and for this implementing a system for specifying sections and weights so that we can have separators between unrelated set of actions and have better control over what appears where.

Reply:
Wow, very nice! This will be a complete solution from the start then. Thanks Andras!